docs/FAQ/how-to-deploy-bold-reports-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,210 @@
+# How to deploy Bold Reports in Elastic Kubernetes Services (EKS) using Application Load Balancer (ALB)
+
+The section explains how to deploy [Bold Reports](https://www.boldreports.com/) in an EKS cluster using an [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+
+1. Create an Amazon EKS cluster and [node group](https://docs.aws.amazon.com/eks/latest/userguide/eks-compute.html) to deploy Bold Reports.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html 
+
+2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following this [link](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html).
+
+   ![AWS EFS](../images/aws-efs.png)
+
+3. Connect to your Amazon EKS cluster.
+   https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/
+  
+4. Deploy the EFS CSI Driver to your cluster.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html 
+
+	```console
+	kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.4"
+	```
+	
+	> <b>Note:</b> Make sure the following network configurations to avoid volume mounting issues.<br/><br/>
+		1. EFS file system in the same region as your EKS cluster.<br/>
+		2. Your EKS cluster and EFS file system must be in the same Virtual Private Cloud (VPC).<br/>
+		3. The security group for your EFS file system, which will allow incoming traffic from your EKS cluster's nodes.<br/>
+		4. Ensure the EFS CSI Driver installation in cluster.<br/>
+		
+5. To install the AWS load balancer controller, please refer to the following documentation.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html
+
+6. Download the **deploy_eks_alb.yaml** file [here](../../deploy/deploy/deploy_eks_alb.yaml) to deloy Bold Reports on EKS.
+
+7. Navigate to the folder where the deployment files were downloaded in above step. 
+
+8. Open the **deploy_eks_alb.yaml** file that was downloaded in Step 6. Replace the file system ID that was created in Step 2 with the `<efs_file_system_id>` placeholder in the file.
+
+   ![PV Claim](./images/alb_efs_file_system.png)
+
+9. Enter the necessary variable information in the **deploy_eks_alb.yaml** file to complete the auto-deployment, as detailed below.
+
+    * Enter the Bold Reports license key, user, and database server details.
+	The following environment variables are optional. If not provided, a manual Application start-up configuration is needed.
+        
+        ![Licence-and-User-Details](../images/deploy/eks/licence-and-user-details.png)
```

Comment:
This is image was broken give proper path.

Reply:
removed the auto deployment related md